ocrd_network/ocrd_network/processing_worker.py

```diff
@@ -79,6 +80,11 @@ def __init__(self, rabbitmq_addr, mongodb_addr, processor_name, ocrd_tool: dict,
         # The publisher is connected when the `result_queue` field of the OcrdProcessingMessage is set for first time
         # Used to publish OcrdResultMessage type message to the queue with name {processor_name}-result
         self.rmq_publisher = None
+        # Always create a queue (idempotent)
+        self.create_queue(
```

Comment:
We then do not need to pass parameters here but use the defaults assigned in the method signature.

Reply:
I committed accordingly.